### PR TITLE
decorator jobs example 2 QAOA

### PR DESCRIPTION
**Pull request by mbeach-aws**
major rewrite of example 2 to simply.

### CODE REVIEW

File: examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "\n",
```
Comment (licedric) [inline examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb:25]: <pre class="ql-syntax" spellcheck="false">Line #2.    print(job.result()["task summary"])
</pre><p>task summary is empty?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (speller26) [inline examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb:25]: <p>This blurb is probably sufficient for all the other examples after the getting started, as opposed to the longer explanation of hybrid jobs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (speller26) [inline examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb:25]: <p>I don't think the Task Summary is needed here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (licedric) [reply examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb:25]: <p>Are there no other examples of creating jobs from scripts that we can link from?</p>

Comment (licedric) [inline examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb:25]: <p>I assume you mean "from a previous"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (licedric) [inline examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb:25]: <p>But how do you load training progress?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

**Pull request merged**